carrot.css

```
@@ -0,0 +1,86 @@
+.sean-whole-part {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #f8f9fa;
+  padding: 60px 0px;
+}
+
+.sean-popular-products {
+  width: 1024px;
+  height: 1088px;
+  margin: 60px, 152px, 120px;
+}
+
+.sean-title {
+  font-size: 40px;
+  line-height: 1.35;
+  font-weight: 700;
+  margin-bottom: 80px;
+  text-align: center;
+}
+
+::selection {
+  color: #212529;
+  background: #ffb98b;
+}
+
+.sean-products-container {
+  display: flex;
+  justify-content: space-around;
+  flex-wrap: wrap;
+  width: 100%;
+}
+
+.sean-product-whole {
+  margin-right: 44px;
+  margin-bottom: 56px;
+}
+
+#sean-product-4 {
+  margin-right: 0px;
+}
+
+#sean-product-8 {
+  margin-right: 0px;
+}
```

```suggestion
.sean-products-container {
  display: flex;
  justify-content: space-around;
  gap: 44px;
  flex-wrap: wrap;
  width: 100%;
}

.sean-product-whole {
  margin-bottom: 56px;
}
```

만약 보여줘야 하는 줄의 수가 계속 바뀐다면 지금 구조로는 그 때 마다 #sean-product-12, #sean-product-16, ... 이렇게 추가해줘야 하겠죠?

그런 일을 방지함과 동시에 코드를 다음과 같이 줄여볼 수 있습니다. display: flex 속성을 준 컨테이너에 gap 속성을 주면 무슨 일이 발생하는지 찾아보길 바랍니다!